lib/pool/BaseConnectionPool.js

```diff
@@ -8,6 +8,7 @@
  *
  */
 
+
```

Comment:
This is another instance of a change that causes hundreds of files to change, maybe not worth it.

Reply:
That was an accidental change due to GREP replace.
There are still gonna be nearly 800 file changes as we have nearly 400 API functions and each has a function definition and a type definition files.